easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.8.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,43 @@
+name = 'TensorFlow'
+version = '1.8.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'TensorFlow-1.5.0_swig-env.patch',
+    'TensorFlow-%(version)s_icc-fixes.patch',
+    'TensorFlow-1.8.0_remove-msse-hardcoding.patch',
+    'TensorFlow-1.6.0_use_64b_alignment.patch',
+    'TensorFlow-1.7.0_remove_DATE_defines.patch',
+    'TensorFlow-1.8.0_no_grpc.patch',
+    'TensorFlow-1.8.0_no_boringssl.patch',
```

Comment:
@akesandgren Please use `%(version)s` rather than `1.8.0` for all patches that are specific to `1.8.0`?

Reply:
Maybe.... not sure when %(version)s is preferable and not.

These patches could probably be reused for the next version.

Reply:
@akesandgren We usually use `%(version)s` if the version in the patch filename matches `version`, that serves as a reminder to re-check the patches when the version is bumped...

Also, please also include `TensorFlow-1.8.0_lrt-flag.patch`, cfr. #6259